### ISSUE TITLE
Feature request: Run FAST.Farm without wind

### ISSUE THREAD

**Issue by lucas-carmo** [OpenFAST/openfast#2566]
**Is your feature request related to a problem? Please describe.**
It would be useful to be able to run FAST.Farm without environmental loads to find the equilibrium position of floating systems with shared moorings. We can disable the waves with `WaveMod=0` in SeaState.dat but we cannot do something similar for the wind when using FAST.Farm.

**Describe the solution you'd like**
It would be nice to have an option `ModWake=0` to skip the wake calculations and `Mod_AmbWind=0` to remove the ambient wind (though the later could be achieved with `WindType=1` and `HWindSpeed=0` in InflowWind).

**Describe alternatives you've considered**
I have tried different combinations of zero/low wind speed with different AeroDyn configurations but they all fail for some reason. For example, I get the following after a few seconds if I try zero wind speed in InflowWind and `Wake_Mod=0` in AeroDyn:

![image](https://github.com/user-attachments/assets/f112f9dc-e99b-4747-b407-c061b2d95c9a)

**Comment by andrew-platt** [comment]
I don't know if we will be able to get this into v 4.0, but we can probably add it to a 4.1 or the 5.0 (whichever comes first).

**Comment by jjonkman** [comment]
I agree that this may be useful feature to add, but it may not be as trivial as suggested given that much of FAST.Farm revolves around coupling the wind turbines of a wind farm together through wind/wakes.

This topic was actually discussed recently on our forum: https://forums.nrel.gov/t/fast-farm-without-wind-modelling-setup/7080, where to avoid the +/- pi/2 error, a workaround was proposed involving:

- Set up the FAST.Farm simulation with a low steady wind speed, low rotor thrust, and no wind turbine control
- To set steady wind speed, use `Mod_AmbWind` = 2 with `WindType` = 1 in InflowWind
- To reduce the rotor thrust, reduce the chord (`BlChord`) and aerodynamic lift and drag coefficients in AeroDyn
- To avoid numerical issues in AeroDyn, set `WakeMod` = 0, `AFAeroMod` = 1, `TwrPotent` = 0, `TwrShadow` = 0, and `TwrAero` = FALSE.
- To disable wind turbine control, set `CompServo` = 0 in OpenFAST
- Set a low value of `NumPlanes` (e.g., NumPlanes = 2) in FAST.Farm to ensure that there are no wake effects between the turbines

Best regards,

**Comment by lucas-carmo** [comment]
Thanks, @jjonkman! I had tried those configurations but did not reduce the chord, lift, and drag coefficients. That should do the trick!

Best,
